RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -880,12 +880,31 @@ boolean is100FamRun()
 	return true;
 }
 
+boolean pathAllowsFamiliar()
```

Comment:
nit:L would prefer if this used `$classes[A, B, C] contains auto_my_class`, and separately had the pokefam condition

$strings[] won't error if you provide it a path name that isn't a valid path

Reply:
Why does pokefam need to be seperate?

Reply:
Change made.
I used my_class() as there is no auto_my_class() function.
Also license to adventure needed to remain in strings form because it does not use a custom class.
It validates now (had to change `Actually Ed the Undying` to `Ed` for that to happen too)

Reply:
change made